net/net-snmp/Makefile

```diff
@@ -209,6 +209,7 @@ define Build/InstallDev
 	$(INSTALL_DIR) $(2)/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/net-snmp-config $(2)/bin/
 	$(SED) 's,=/usr,=$(STAGING_DIR)/usr,g' $(2)/bin/net-snmp-config
+	ln -sf $(STAGING_DIR)/host/bin/net-snmp-config $(STAGING_DIR)/usr/bin/
```

Comment:
Using `$(STAGING_DIR)/host` is deprecated in favour of `$(STAGING_DIR_HOSTPKG)`. Please use that instead.